src/main/webapp/login/loginModal.js

```diff
@@ -5,7 +5,7 @@ import Container from './container'
 export const LogInModal = props => {
   const { handleClose, open, label } = props
   return (
-    <Dialog open={open} onClose={handleClose}>
```

Comment:
What is this change for?

Reply:
This is so that the modal doesn't flicker when you hit `esc` or click off of the modal

Reply:
ohh, cause only the container should be able to close the modal.